app/assets/javascripts/to_do_lists.coffee

```diff
@@ -1,3 +1,119 @@
 # Place all the behaviors and hooks related to the matching controller here.
 # All this logic will automatically be available in application.js.
 # You can use CoffeeScript in this file: http://coffeescript.org/
+
+class @TodoList
+  initialize: () ->
+    bindAddEvent()
+
+  onCreate: (options) ->
+    if options.error
+      showCheckAlert detectCreateListAlertMessage()
+    else
+      closePopUp()
+      appendNewOne options.html
+      scrollToNewOne options.id
+      bindAddItemEvent options.id
+
+  detectAddButton = () ->
+    $('#add-list-button')
+
+  detectContainer = () ->
+    $('div.container')
+
+  detectPopUp = () ->
+    $('#add-list-modal')
+
+  detectList = (id) ->
+    $("#list-#{id}")
+
+  detectNewListSubmitButton = () ->
+    $('#new-list-submit-button')
+
+  detectNewListTitle = () ->
+    $('#list_title')
+
+  detectNewListDescription = () ->
+    $('#list_description')
+
+  detectNewListCloseButton = () ->
+    $('#close-new-list-form')
+
+  detectNewListAlertMessage = () ->
+    $('#alert-box-new-list')
+
+  detectCreateListAlertMessage = () ->
+    $('#alert-box-create-list')
+
+  appendNewOne = (html) ->
+    detectContainer().append html
+
+  showCheckAlert = (alertBox) ->
+    if alertBox.css('display') == 'none'
+      alertBox.fadeIn 'slow'
+      alertBox.delay 3000
+      alertBox.fadeOut 'slow'
```

Comment:
@sergeyradkevich, I don't really think it can be used here, as I have only one condition and three blocks of code to be executed like entire entity. The only way I see is to extract it in separate method and then refactor method like this:
```javascript
displayAlertBox = (alertBox) ->
    alertBox.fadeIn('slow').delay(3000).fadeOut('slow')

  showCheckAlert = (alertBox) ->
    displayAlertBox(alertBox) if alertBox.css('display') == 'none'
```